### PR TITLE
nautilus: mgr/insights: Test environment requires 'six'

### PR DESCRIPTION
**Pull request by badone**
This is causing nautilus 'make check' builds to fail in Jenkins with "E   ModuleNotFoundError: No module named 'six'" when running 'build/insights/py3/bin/py.test tests/'

Example: https://jenkins.ceph.com/job/ceph-pull-requests/64761/consoleFull#1181206387c19247c4-fcb7-4c61-9a5d-7e2b9731c678

Signed-off-by: Brad Hubbard <bhubbard@redhat.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

File: src/pybind/mgr/insights/tox.ini [hunk 1]
```diff
@@ -8,6 +8,7 @@ minversion = 2.8.1
 deps =
     pytest
     mock
+    six>=1.14.0
```
Comment (tchaikov) [inline src/pybind/mgr/insights/tox.ini:11]: might want to note down why this change is not cherry-picked from master, i think it's likely due to that master has dropped the python2 support. see also 6bdb280954bda42cc8de37165a368cc30450f858

**Review by tchaikov — APPROVED** [review]

File: src/pybind/mgr/insights/tox.ini (hunk 1 above)
Comment (badone) [reply src/pybind/mgr/insights/tox.ini:11]: ack

**Comment by smithfarm** [comment]
https://tracker.ceph.com/issues/48425
